flask_caching/backends/base.py

```diff
@@ -13,7 +10,7 @@ def iteritems_wrapper(mappingorseq):
 
     """
     if hasattr(mappingorseq, "items"):
-        return iteritems(mappingorseq)
+        return iter(mappingorseq.items())
```

Comment:
In Python 3, items() returns an iterator and not a list --> ``iter`` is not needed here

Reply:
Yes, that was suspicious for me, i just couldn’t wrap my head around it early in the morning i guess.